scripts/kind-e2e/lib_armada_deploy_kind.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# This should only be sourced
+if [ "${0##*/}" = "lib_helm_deploy_subm.sh" ]; then
+    echo "Don't run me, source me" >&2
+    exit 1
+fi
+
+### Variables ###
+
+# FIXME: Armada support for setting pod/service CIRDs via flags?
+declare -A cluster_CIDRs=( ["cluster1"]="10.4.0.0/14" ["cluster2"]="10.8.0.0/14" ["cluster3"]="10.12.0.0/14" )
+declare -A service_CIDRs=( ["cluster1"]="100.1.0.0/16" ["cluster2"]="100.2.0.0/16" ["cluster3"]="100.3.0.0/16" )
+
+kubecfgs_rel_dir=scripts/output/kube-config/container/
+kubecfgs_dir=${PRJ_ROOT}/$kubecfgs_rel_dir
+
+### Functions ###
+
+function install_armada() {
+    test -x /go/bin/armada && return
+    # FIXME: Use a release from GitHub once one is available
+    curl -L https://drive.google.com/u/0/uc\?id\=17r6E0RwbHcvlaFl0LKFICuAc_eGb13W9\&export\=download \
+         -o /go/bin/armada
+    chmod a+x /go/bin/armada
+    /go/bin/armada version
+}
+
+function create_kind_clusters() {
+    install_armada
+
+    # Armada -n basically adds clusters to make sure there are at least n
+    /go/bin/armada create clusters -n 1 --tiller
```

Comment:
I think we only need tiller when deploying with helm.

Reply:
Fixed in the latest (and #357 added to test), thanks!